app/src/components/searchWidget/date-input/expandedDatePanel.tsx

```diff
@@ -66,37 +71,53 @@ export class ExpandedDatePanel extends React.Component<StateChangedProps, any> {
                 unknownDates: this.props.initialModel.uncertainDates
             });
         }
-
         this.props.onChange(this.props.initialModel);
     }
 
     saveSelectedPanel(panelType: datePanelTypes) {
         if (panelType === datePanelTypes.UNKNOWN_DATES) {
             this.props.initialModel.dateFrom = null;
             this.props.initialModel.dateUntil = null;
+            this.props.initialModel.clicked = false;
+            this.props.initialModel.clickedu = true;
         } else if (panelType === datePanelTypes.SPECIFIC_DATES) {
             this.props.initialModel.uncertainDates = null;
+            this.props.initialModel.clickedu = false;
+            this.props.initialModel.clicked = true;
         }
     }
 
     render() {
+        //let btn_class = this.state.oldButton ? "blackButton" : "whiteButton";
         const { from, to } = this.state;
         return (
             <div className="date-panel-expanded">
-                <div className="top-toogler">
+                <ToggleButtonGroup className="datetoggle" name="options" defaultValue={1}>
                     <div
                         className="dates-selector middle-text"
                         onClick={() => this.selectPanel(datePanelTypes.SPECIFIC_DATES)}
                     >
-                        <div>Specific Dates</div>
+                        <Button
+                            active={this.props.initialModel.clicked}
+                            className={'btn1'}
+                            onClick={() => this.saveSelectedPanel(datePanelTypes.SPECIFIC_DATES)}
+                        >
+                            Specific Dates
+                        </Button>
                     </div>
                     <div
                         className="dates-selector middle-text"
                         onClick={() => this.selectPanel(datePanelTypes.UNKNOWN_DATES)}
                     >
-                        <div>Flexible Dates</div>
+                        <Button
+                            active={this.props.initialModel.clickedu}
```

Comment:
active={!this.props.initialModel.unknownDates}

so REMOVE clicked/clickedu states